javascript/sdk/src/repl.ts

```diff
@@ -1,5 +1,10 @@
 import { ExecResponse } from './types'
 
+export interface ExecOptions {
+  timeout?: number
```

Comment:
let's put units here (`timeout_ms`, or maybe we should switch to `timeout_seconds` universally -- I don't know if ms is the right unit here)